### PR TITLE
Add Marketplace and connections documentation

### PR DESCRIPTION
**Pull request by shaniR**

### CODE REVIEW

File: en/docs/choreo-concepts/choreo-marketplace.md [hunk 1]
```diff
@@ -1,52 +1,69 @@
...
+### Add a Choreo service
 
-![Filter by price](../assets/img/choreo-concepts/marketplace/filter-by-price.png){.cInlineImage-full}
+In Choreo, a service exposed through the platform is termed a Choreo service, with each service being identifiable by an endpoint within a Choreo service component. The Marketplace showcases a service for each endpoint within a service component.
+
+Upon deployment to the initial environment, services get automatically added to the Marketplace. Choreo effortlessly collects essential details such as component name, endpoint name, description, and service definitions during this deployment, utilizing them to generate the corresponding service entries in the Marketplace.
+
+The service name follows the convention of `component name - endpoint name`, while all other details remain unchanged.
+
+## Edit services in the marketplace
```
Comment (Nashaath) [inline en/docs/choreo-concepts/choreo-marketplace.md:57]: Need to capitalize M in the term `Marketplace`.

File: en/docs/develop-components/sharing-and-reusing-services.md [hunk 2]
```diff
@@ -1,36 +1,53 @@
...
+Now you have created a connection to the service. Let's see how you can consume it. 
 
-Refer to the [Private Package Usage](https://github.com/wso2/choreo-sample-apps/tree/main/ballerina/private-package-usage) sample to learn more about using Ballerina private packages in Choreo.
+## Consuming a service through a connection
+
+You can consume a Choreo-deployed service within another service
+
+Consuming connections from within Choreo services is seamless and straightforward. Follow the steps below to consume a Choreo service. 
+
+### Consuming services via the Choreo console
```
Comment (Nashaath) [inline en/docs/develop-components/sharing-and-reusing-services.md:33]:
```suggestion
### Consuming services via the Choreo Console
```

File: en/docs/develop-components/sharing-and-reusing-services.md [hunk 3]
```diff
@@ -1,36 +1,53 @@
...
+
+You can consume a Choreo service via another service you plan to deploy in Choreo as follows: 
+
+1. In the Choreo console navigate to the project or component to view the connections in the respective visibility level.
```
Comment (Nashaath) [inline en/docs/develop-components/sharing-and-reusing-services.md:37]:
```suggestion
1. In the Choreo Console, navigate to the project or component to view the connections in the respective visibility level.
```

File: en/docs/develop-components/sharing-and-reusing-services.md [hunk 4]
```diff
@@ -1,36 +1,53 @@
...
+3. Click on the service you wish to consume. 
+4. Refer to the **Adding Connection Configuration to Application** section in your component-config file. 
+5. Replace the placeholder <YOUR_ENVIRONMENT_VAR_HERE> with the environment variable names. When you deploy your component, the configured environment variables will be available with their values. Refer to the service's **Read configurations** section (exploring the service via the Choreo marketplace) to learn how to read these environment variables and use them in your code.  
+6. Refer to the entire **Developer Guide** pane to learn how to obtain a token and consume the service.
+
+### Consuming Choreo services via external applications
+
+You can consume a service deployed on Choreo via an external application. To consume a Choreo deployed service you need to create a project-level connection. You can then consume the connection as follows: 
+
+1. In the Choreo console, select the project to view the connections in the respective visibility level.
```
Comment (Nashaath) [inline en/docs/develop-components/sharing-and-reusing-services.md:48]:
```suggestion
1. In the Choreo Console, select the project to view the connections in the respective visibility level.
```

File: en/docs/choreo-concepts/connections.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+Services can exist in two main forms: standalone and integrated. By connecting services, organizations can access and benefit from the combined strength of various features and capabilities. Choreo allows you to connect services using Connections. 
+
+Connections in Choreo allows you to integrate Choreo components, or to integrate Choreo components to external services or resources. Connections provide a simple and uniform way to integrate with services and resources. A Connection will include the set of parameters that the application requires to create a connection at runtime. The visibility level of the Connection and the service for which it is created determine these parameters. You can easily refer to these parameters in the application runtime.
+ 
+You can add Connections in different visibility levels: Organization, Project, and Component. The visibility levels are described below:
+
+## Organization Connections
+
+Organization Connections are Connections created for services within a particular organization. These Connections **can be used by any component within the organization**.
```
Comment (Nashaath) [inline en/docs/choreo-concepts/connections.md:11]: Would it make sense to say:
"Organization Connections are Connections you create to connect to services within a particular organization."

File: en/docs/choreo-concepts/connections.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+For example, create an Organization Connection to share a database across the organization for all the projects and components to reuse. Components can refer to Organization Connections using the connection id. 
+Organization connections created to consume Choreo services under the OAuth security scheme will have their own OAuth application per connection. Any component reusing such a connection will use the same client id and client secret.
+
+## Project Connections
+
+Project Connections are Connections created for Services at the project level. The Connections **can be used by any component within the project**.
```
Comment (Nashaath) [inline en/docs/choreo-concepts/connections.md:18]: Would it make sense to say:
Project Connections are Connections you create to connect to services within a particular project.

File: en/docs/choreo-concepts/choreo-marketplace.md [hunk 7]
```diff
@@ -1,52 +1,59 @@
...
+You can click on the service card to open the detailed view of the service. The detailed service page features the service name, summary, version, labels, and service icon as the header.
 
-![Connector details](../assets/img/choreo-concepts/marketplace/connector-details.png)
+Choreo organizes the service content into four tabs. The four tabs contain information as follows: 
 
-If you want to view more information about a specific connector, click on it. This directs you to an overview page with details about versions, compatibility, and links for reporting issues, accessing API documentation, contacting the vendor, and more.
+- **Overview**: Choreo displays the service overview provided by the service developer. If the service developer has not provided any content at service creation, this section will be disabled. The service developer can provide the overview content via the Manage → Marketplace section of the component.
 
-## Filter by category
+- **API definition**: Includes the service's API definition, extracted from the user repository using the component-config file or endpoints.yaml.
```
Comment (Nashaath) [inline en/docs/choreo-concepts/choreo-marketplace.md:38]:
```suggestion
- **API definition**: Includes the service's API definition, extracted from the user repository using the `component-config` file or `endpoints.yaml` file. 
```

File: en/docs/choreo-concepts/connections.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+Connections in Choreo allows you to integrate Choreo components, or to integrate Choreo components to external services or resources. Connections provide a simple and uniform way to integrate with services and resources. A Connection will include the set of parameters that the application requires to create a connection at runtime. The visibility level of the Connection and the service for which it is created determine these parameters. You can easily refer to these parameters in the application runtime.
+ 
+You can add Connections in different visibility levels: Organization, Project, and Component. The visibility levels are described below:
+
+## Organization Connections
+
+Organization Connections are Connections created for services within a particular organization. These Connections **can be used by any component within the organization**.
+
+For example, create an Organization Connection to share a database across the organization for all the projects and components to reuse. Components can refer to Organization Connections using the connection id.
```
Comment (Nashaath) [inline en/docs/choreo-concepts/connections.md:13]: Shall we replace all occurrences of id with ID based on https://developers.google.com/style/word-list#id

File: en/docs/choreo-concepts/connections.md (hunk 5 above)
Comment (shaniR) [reply en/docs/choreo-concepts/connections.md]: done

File: en/docs/choreo-concepts/connections.md (hunk 8 above)
Comment (shaniR) [reply en/docs/choreo-concepts/connections.md]: done

File: en/docs/choreo-concepts/connections.md (hunk 6 above)
Comment (shaniR) [reply en/docs/choreo-concepts/connections.md]: done

**Review by Nashaath — APPROVED** [review]

**Pull request merged**